stdlib/public/core/CMakeLists.txt

```diff
@@ -227,12 +227,7 @@ else()
   # effort has been completed.
   #set(LINK_FLAGS
   #  -Wl,--whole-archive swiftRuntime -Wl,--no-whole-archive)
-  if("${SWIFT_PATH_TO_LIBICU_BUILD}" STREQUAL "")
-    list(APPEND swift_core_private_link_libraries
-      ICU_UC ICU_I18N)
-  else()
-    list(APPEND swift_core_private_link_libraries -licui18n -licuuc -licudata)
-  endif()
+  list(APPEND swift_core_private_link_libraries ICU_UC ICU_I18N)
 endif()
```

Comment:
I don't think `ICU_UC` and `ICU_I18N` are actually the correct variables I think it should be `ICU_UC_LIBRARIES` and `ICU_I18N_LIBRARIES`

Reply:
Similar to my explanation above, this is just following the previous code. The code uses those values (they are not variables) as sigils for linking the ICU libraries. When those sigils are found in `AddSwift.cmake`, the right thing is done: for most SDKs `${SWIFT_${sdk}_${arch}_ICU_UC}` and similar are used, but if one provided `${SWIFT_PATH_TO_LIBICU_BUILD}`, the normal `-licuuc` flags are used (since the path will be in the library search path).